### DISCUSSION TITLE
TabItem:pointerover doesn't change Background fully.

### DISCUSSION THREAD

**Discussion by BytePetter** [AvaloniaUI/Avalonia#12003] [Q&A]
**Describe the bug**
When having a style defined for a TabItem that is pointed over like this:

`<Style Selector="TabItem:pointerover /template/ ContentPresenter#PART_ContentPresenter">
	<Setter Property="Background" Value="#blue" />
	<Setter Property="Foreground" Value="#FFFFFF" />
</Style>`

it does NOT paint the entire background of the TabItem in a TabControl whichs TabStripPlacement is 'Left' in this color, but only the small Text it displays.

**To Reproduce**
Simply produce a TabControl and style the TabItems like described above.

**Expected behavior**
The entire TabItem painted in the defined Background, as it does correctly when NOT defining the metaclass :pointerover.

**Screenshots**
![image](https://github.com/AvaloniaUI/Avalonia/assets/29927039/09062fc5-6c63-4e2a-8066-acf5a7b6f128)

An example with 3 TabItems
The first is selected.
The second which is pointedover, styled to have Background=blue
The third which is not pointedover, styled to have Background=red

**Desktop (please complete the following information):**

OS: Windows 10

**Comment by timunie** [comment]
Please lookup the original style and also use DevTools via F12 to check visual tree. 

If you are using 0.10.x, remember to switch the branch.

**Reply by Tar0k** [reply]
The same problem, but with the button.

**Reply by timunie** [reply]
the same solution, lookup original styles in source and check out DevTools [F12]

**Reply by Tar0k** [reply]
Thank you. I found a problem. The problem was with Suki theme. My fault was to use custom theme without knowledge how to use it.

**Comment by mayka-mack** [comment]
@BytePetter, you've likely found a solution by now, but in case it helps anyone else, I experienced the same behavior when using Avalonia's Fluent Theme.

To color the **whole** background for a `TabItem` which is pointed over, I needed to target `Border#PART_LayoutRoot`.
```
<Style Selector="TabItem:pointerover /template/ Border#PART_LayoutRoot">
    <Setter Property="Background" Value="blue" />
</Style>
<Style Selector="TabItem:pointerover /template/ ContentPresenter#PART_ContentPresenter">
    <Setter Property="Foreground" Value="#FFFFFF" />
</Style>
```

Side note: if you want to also hide the pipe line for a selected `TabItem`, add this as well:
```
<Style Selector="TabItem:selected /template/ Border#PART_SelectedPipe">
    <Setter Property="IsVisible" Value="false" />
</Style>
```

**Comment by begleysm** [comment]
> @BytePetter, you've likely found a solution by now, but in case it helps anyone else, I experienced the same behavior when using Avalonia's Fluent Theme.
> 
> To color the **whole** background for a `TabItem` which is pointed over, I needed to target `Border#PART_LayoutRoot`.
> 
> ```
> <Style Selector="TabItem:pointerover /template/ Border#PART_LayoutRoot">
>     <Setter Property="Background" Value="blue" />
> </Style>
> <Style Selector="TabItem:pointerover /template/ ContentPresenter#PART_ContentPresenter">
>     <Setter Property="Foreground" Value="#FFFFFF" />
> </Style>
> ```
> 
> Side note: if you want to also hide the pipe line for a selected `TabItem`, add this as well:
> 
> ```
> <Style Selector="TabItem:selected /template/ Border#PART_SelectedPipe">
>     <Setter Property="IsVisible" Value="false" />
> </Style>
> ```

@mayka-mack Thank you for the above.  It has been quite helpful.

Do you know of a way to NOT have the background change when mousing over a TabItem Header?  Either disabling it or setting it to it's previous value (either the Selected color or the non-Selected color)?
